django/db/backends/sqlite3/schema.py

```diff
@@ -47,6 +47,12 @@ def _remake_table(self, model, create_fields=[], delete_fields=[], alter_fields=
                       override_indexes=None):
         """
         Shortcut to transform a model from old_model into new_model
+
```

Comment:
Bit of detail about the schema editor changes here: https://groups.google.com/forum/#!topic/django-developers/KTmiKneaGXE

I'll move this into its own PR.